### PR TITLE
feat: fetch applications using grpc

### PR DESCRIPTION
**Pull request by esoubiran-aneo**
fix: https://github.com/aneoconsulting/ArmoniK/issues/671
fix: https://github.com/aneoconsulting/ArmoniK/issues/673
fix: https://github.com/aneoconsulting/ArmoniK/issues/674
fix: https://github.com/aneoconsulting/ArmoniK/issues/666
fix https://github.com/aneoconsulting/ArmoniK/issues/691

### CODE REVIEW

File: apps/app/src/app/core/core.module.ts [hunk 1]
```diff
@@ -10,18 +10,20 @@ import {
...
-  FavoritesService,
-  GrafanaService,
   LanguageService,
   PagerService,
   SeqService,
   SessionsService,
   SettingsService,
   TasksService,
 } from './services';
+import {} from './services/grpc/grpc-applications.service';
```
Comment (maurelleAneo) [inline apps/app/src/app/core/core.module.ts:26]: Mal exporté ici

File: apps/app/src/app/pages/pages-routing.module.ts [hunk 2]
```diff
@@ -24,6 +24,13 @@ const routes: Routes = [
             (m) => m.SessionsModule
           ),
       },
+      {
+        path: 'applications-2',
```
Comment (maurelleAneo) [inline apps/app/src/app/pages/pages-routing.module.ts:28]: Pourquoi ce nom ?

File: apps/app/src/app/pages/pages/applications-list/applications-list.component.html [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+      [clrDgSortOrder]="defaultSortOrder(OrderByField.ORDER_BY_FIELD_SERVICE)"
+    >
+      {{ 'pages.applications-list.columns.service' | translate }}
+    </clr-dg-column>
+    <clr-dg-column>
+      {{ 'pages.applications-list.columns.actions' | translate }}
+    </clr-dg-column>
+
+    <ng-container *ngIf="loadApplications$ | async as list">
+      <clr-dg-row *ngFor="let application of list.application">
```
Comment (maurelleAneo) [inline apps/app/src/app/pages/pages/applications-list/applications-list.component.html:86]: TrackBy manquant

File: apps/app/src/app/pages/pages/applications-list/applications-list.component.ts [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+  private _state: ClrDatagridStateInterface = {};
+
+  private _subjectManual = new Subject<void>();
+  private _subjectDatagrid = new Subject<ClrDatagridStateInterface>();
+  private _subjectInterval = new BehaviorSubject<number>(this.initialInterval);
+  private _subjectStopInterval = new Subject<void>();
+
+  /** Triggers to reload data */
+  private _triggerManual$ = this._subjectManual.asObservable();
+  private _triggerDatagrid$ = this._subjectDatagrid.asObservable().pipe(
```
Comment (maurelleAneo) [inline apps/app/src/app/pages/pages/applications-list/applications-list.component.ts:46]: Pareil ici

File: apps/app/src/assets/i18n/fr.json [hunk 5]
```diff
@@ -43,6 +44,32 @@
...
+            "title": "Rafraîchissement automatique",
+            "disabled": "Désactivé"
+          },
+          "disable": "Désactiver"
+        },
+        "related_sessions": "Sessions liées"
+      },
+      "pagination": {
+        "per_page": "Par page",
+        "applications": "{{value}} résultats",
```
Comment (maurelleAneo) [inline apps/app/src/assets/i18n/fr.json:69]: `applications`

File: apps/app/src/assets/i18n/fr.json [hunk 6]
```diff
@@ -43,6 +44,32 @@
...
+        "application": "{{value}} résultat"
```
Comment (maurelleAneo) [inline apps/app/src/assets/i18n/fr.json:70]: `application`

**Review by maurelleAneo — CHANGES_REQUESTED** [review]

File: apps/app/src/app/core/services/grpc/grpc-applications.service.ts [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+import { mergeMap, Observable, takeUntil, throwError, timer } from 'rxjs';
+import { GrpcParams } from '../../types/grpc-params.type';
+import {
+  ListApplicationsRequest,
+  ListApplicationsResponse,
+} from '../../types/proto/applications-common.pb';
+import { ApplicationsClient } from '../../types/proto/applications-service.pbsc';
+
+@Injectable()
+export class GrpcApplicationsService {
```
Comment (esoubiran-aneo) [inline apps/app/src/app/core/services/grpc/grpc-applications.service.ts:11]: Il faudra utiliser le BaseGrpcService

**Pull request closed without merging**